Description: make lxc-shutdown -h work
 It was calling non-existent 'help'
Author: Timothy Chen <tnachen@gmail.com>
Forwarded: yes
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/980905

Index: lxc-0.7.5/src/lxc/lxc-shutdown.in
===================================================================
--- lxc-0.7.5.orig/src/lxc/lxc-shutdown.in	2012-04-16 11:57:48.000000000 -0500
+++ lxc-0.7.5/src/lxc/lxc-shutdown.in	2012-04-16 11:59:11.158129013 -0500
@@ -60,7 +60,7 @@
 while true; do
     case "$1" in
     -h|--help)
-        help
+        usage
         exit 1
         ;;
     -n|--name)
